plugins/content/caption/base/services/CaptionAssetService.php

```diff
@@ -209,7 +215,29 @@ protected function attachFile(CaptionAsset $captionAsset, $fullPath, $copyOnly =
 		$captionAsset->setStatus(CaptionAsset::ASSET_STATUS_READY);
 		$captionAsset->save();
 	}
-    
+
+	/**
+	 * @param CaptionAsset $captionAsset
+	 * @param string $path
+	 */
+	protected function detectMultiLanguageFile(CaptionAsset $captionAsset, $path = null)

```

Comment:
@shpiNi  - I'd change the name to something like isMultiLanguageFile. It implies on the expected returned value more clearly.